### ISSUE TITLE
Example for acutal data of type "Map"

### ISSUE THREAD

**Issue by yphuangms** [Azure/opendigitaltwins-dtdl#24]
When define a "Property" of type "Map", how do we actually assigned / formatted value for this property? Is there any example?

```
{
    "@type": "Property",
    "name": "modules",
    "writable": true,
    "schema": {
        "@type": "Map",
        "mapKey": {
            "name": "moduleName",
            "schema": "string"
        },
        "mapValue": {
            "name": "moduleState",
            "schema": "string"
        }
    }
}

```
Does it look like this:
```
"modules" : [  
      { "moduleName" : "moduel1", "moduleState" : "on" }, 
      { "moduelName" : "module2", "moduleState" : "off" } 
]
```

Or looks like this:
```
"modules" : { 
      "moduel1" : "on",
      "module2" : "off"
}
```

**Comment by askpatrickw** [comment]
Hey @briancr-ms, since it been 4 months, why not put an example in this Issue so there's a clear answer (and maybe someone can do a doc PR for you). ;-)

**Comment by briancr-ms** [comment]
Fixed in #70.
